### ISSUE TITLE
Budgie Places Applet Malfunction

### ISSUE THREAD

**Issue by whatisthiswtf** [budgie-desktop/budgie-desktop#792]
**Operating System**: Elementary OS Loki (Up-to-date)
**Budgie version**: 10.2.9-2~xenial (and earlier)

**The issue encountered**

The "Places" applet in the system tray has malfunctioned. Now clicking on the applet loads a list of available bookmarked locations which work fine; but it also loads false locations with the name of the bookmarked locations written twice which don't do anything once clicked. I've included a screenshot.

When it happened I tried `sudo apt-get purge budgie-desktop` and `sudo apt-get install budgie-desktop`, but it didn't help.

http://i.imgur.com/i02Pk1y.png

**Steps to reproduce the issue**

Not sure how it happened. I was configuring the various settings, adding, removing applets (perhaps more than once) from Raven.

**Comment by feskyde** [comment]
Can you paste here the contents of the `~/.config/gtk-3.0/bookmarks` file?

**Comment by whatisthiswtf** [comment]
@feskyde 

file:///home/this/Documents Documents
file:///home/this/Music Music
file:///home/this/Pictures Pictures
file:///home/this/Videos Videos
file:///home/this/Downloads Downloads
file:///media/this/Windows/Users/whati whati

**Comment by feskyde** [comment]
Try removing the second name from the file (Documents, Music...)

**Comment by whatisthiswtf** [comment]
@feskyde Worked like a charm.

**Comment by whatisthiswtf** [comment]
@feskyde The problem returned once I bookmarked another folder.

**Comment by whatisthiswtf** [comment]
@feskyde I also noticed the "settings.ini" file reads `gtk-application-prefer-dark-theme=0` even though I have it active in Raven. I should also mention I first installed budgie around the time I was installing the Arc theme and loaded it using elementary tweaks (or ubuntu tweaks can't remember) from pantheon. Then after I loaded budgie, I was changing various theme settings from Raven to test out the different configurations till it crashed and it stopped loading the themes correctly. If I remember correctly, I had to boot to a prior kernel to remove budgie (Not to sure about this, maybe ~80%)

**Comment by ikeydoherty** [comment]
```
gtk-application-prefer-dark-theme=0
```

This is the global GTK setting, not the Budgie dark theme mode

**Comment by ikeydoherty** [comment]
> I've also installed f.lux light filter, indicator-cpufreq, indicator-kdeconnect 

Budgie doesn't support indicators. Those you need to take up with @fossfreedom for the indicator applet

**Comment by fossfreedom** [comment]
aye - @whatisthiswtf raise an issue here https://github.com/UbuntuBudgie/budgie-indicator-applet ... but do check that you have actually added the application indicator applet to the panel first!

**Comment by ikeydoherty** [comment]
Lol good point - I should've said that xD

**Comment by whatisthiswtf** [comment]
@fossfreedom @ikeydoherty Thanks! Everything worked well for the application indicator.

**Comment by feskyde** [comment]
> @feskyde The problem returned once I bookmarked another folder.

What file manager are you using?

**Comment by whatisthiswtf** [comment]
@feskyde Pantheon-files

**Comment by feskyde** [comment]
Possibly Pantheon Files is doing something weird with the bookmarks file, raise a bug in the Pantheon Files page

**Comment by whatisthiswtf** [comment]
@feskyde How should I describe the bug?

**Comment by whatisthiswtf** [comment]
@feskyde I thought about this again and I think the second name repeated is the custom folder name as it appears in the bookmarks section in the left pane on Pantheon-files. And the url is the actual path which doesn't change the actual folder name. For example, I added a new bookmark for a folder named Test1 in my home dir, and on the left pane I renamed the bookmarked folder to: "Test1 Custom" so that's how it shows up there. And now the bookmarks file in gtk-3.0 has this line added to the bottom:

    file:///home/this/Test1 Test1 Custom

I also checked with Pantheon-Files on launchpad and they mentioned this most likely has to do with the way Budgie-Desktop reads the bookmarks file.
